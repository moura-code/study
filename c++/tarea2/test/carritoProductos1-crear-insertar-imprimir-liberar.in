crearCarritoProductos
imprimirCarritoProductos
liberarCarritoProductos
crearCarritoProductos
crearFecha 6/6/2001
crearProducto 7 Leche 17
agregarCarritoProductos
imprimirCarritoProductos
liberarCarritoProductos
crearCarritoProductos
crearFecha 1/1/2001
crearProducto 12 Carne 17
agregarCarritoProductos
crearFecha 1/1/2001
crearProducto 10 Lechuga 23
agregarCarritoProductos
crearFecha 1/2/2001
crearProducto 6 Television 23
agregarCarritoProductos
crearFecha 1/1/2003
crearProducto 2 Tomate 23
agregarCarritoProductos
crearFecha 2/1/2001
crearProducto 16 Cafetera 23
agregarCarritoProductos
imprimirCarritoProductos
liberarCarritoProductos
Fin
