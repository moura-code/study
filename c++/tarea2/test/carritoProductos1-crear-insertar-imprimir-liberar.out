Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> Carrito de productos creado de forma exitosa.
2> 3> Carrito de productos liberado con exito.
4> Carrito de productos creado de forma exitosa.
5> Fecha creada en forma exitosa.
6> Producto creado en forma exitosa.
7> Producto agregado de forma exitosa.
8> Producto: 7
Leche
Precio: 17
Ingresado el: 6/6/2001
9> Carrito de productos liberado con exito.
10> Carrito de productos creado de forma exitosa.
11> Fecha creada en forma exitosa.
12> Producto creado en forma exitosa.
13> Producto agregado de forma exitosa.
14> Fecha creada en forma exitosa.
15> Producto creado en forma exitosa.
16> Producto agregado de forma exitosa.
17> Fecha creada en forma exitosa.
18> Producto creado en forma exitosa.
19> Producto agregado de forma exitosa.
20> Fecha creada en forma exitosa.
21> Producto creado en forma exitosa.
22> Producto agregado de forma exitosa.
23> Fecha creada en forma exitosa.
24> Producto creado en forma exitosa.
25> Producto agregado de forma exitosa.
26> Producto: 2
Tomate
Precio: 23
Ingresado el: 1/1/2003
Producto: 6
Television
Precio: 23
Ingresado el: 1/2/2001
Producto: 10
Lechuga
Precio: 23
Ingresado el: 1/1/2001
Producto: 12
Carne
Precio: 17
Ingresado el: 1/1/2001
Producto: 16
Cafetera
Precio: 23
Ingresado el: 2/1/2001
27> Carrito de productos liberado con exito.
28> Fin.
